pkg/domain/cloud.go

```diff
@@ -12,7 +15,40 @@ type CloudAssetChanges struct {
 	AccountID    string
 	Region       string
 	ResourceID   string
-	Tags         map[string]string
+	ARN          string
+	Tags         InboundResourceTags
```

Comment:
Didn't we determined this would be a flat string->string map?

Reply:
Yes but in order for marshalling/unmarshalling, you need a type so you can implement functions for the sql lib to call

Reply:
Okay. In that case we should probably keep that encapsulated somewhere in the data layer/storage package and no in the model used between packages.

Reply:
I'd have to make a data model for the DB INSERT, which will be very redundant with the existing data model, but I can do it for encapsulation

Reply:
I don't think you need to remake the whole model, do you? just a new type you can convert the tags into

Reply:
Maybe but that new type would be outside of the domain package, yet still referenced from within this struct, which is in the domain package.  Me encapsulation would start leakin

Reply:
Actually I'll remove this and change it back to `map[string]string`.  I can encapsulate the conversion in the DB struct functions, which is probably where it really belongs anyway.  (this addresses your next comment too)